.circleci/config.yml

```diff
@@ -30,6 +31,12 @@ references:
         sudo pip3 install awscli
         $(aws ecr get-login --region eu-west-1 --no-include-email)
 
+  install_clamav: &install_clamav
+    run:
+      name: Install Clamav Client
+      command: |
```

Comment:
Do you need the pipe if there is only one command?
```yml
command: sudo apt-get install clamav-daemon -y
```

Reply:
good spot